conf/igenomes.config

```diff
@@ -44,8 +45,25 @@ params {
       intervals              = "${params.igenomes_base}/Homo_sapiens/GATK/GRCh38/Annotation/intervals/wgs_calling_regions.hg38.bed"
       knownIndels            = "${params.igenomes_base}/Homo_sapiens/GATK/GRCh38/Annotation/GATKBundle/{Mills_and_1000G_gold_standard.indels.hg38,beta/Homo_sapiens_assembly38.known_indels}.vcf.gz"
       knownIndelsIndex       = "${params.igenomes_base}/Homo_sapiens/GATK/GRCh38/Annotation/GATKBundle/{Mills_and_1000G_gold_standard.indels.hg38,beta/Homo_sapiens_assembly38.known_indels}.vcf.gz.tbi"
-      snpeffDb               = "GRCh38.86"
-      vepCacheVersion        = "95"
+      snpeffDb               = 'GRCh38.86'
+      vepCacheVersion        = '95'
+      species                = 'homo_sapiens'
+    }
+    'GRCm38' {
+      bwaIndex               = "${params.igenomes_base}/Mus_musculus/Ensembl/GRCm38/Sequence/BWAIndex/genome.fa.{amb,ann,bwt,pac,sa}"
+      chrDir                 = "${params.igenomes_base}/Mus_musculus/Ensembl/GRCm38/Sequence/Chromosomes"
+      chrLength              = "${params.igenomes_base}/Mus_musculus/Ensembl/GRCm38/Sequence/Length/GRCm38.len"
+      dbsnp                  = "${params.igenomes_base}/Mus_musculus/Ensembl/GRCm38/MouseGenomeProject/Annotation/mgp.v5.merged.snps_all.dbSNP142.vcf.gz"
```

Comment:
Where did you download these files from  @apeltzer ? Looks like they are the latest but may be worth documenting that somewhere e.g.

`dbSNP` files for `GRCm38` were downloaded from `ftp://ftp-mouse.sanger.ac.uk/REL-1505-SNPs_Indels/` on [date].

Reply:
These files are quite difficult to find so worth being a bit more transparent whilst things are fresh!

Reply:
I've added a PR to AWS-iGenomes where I'm adding these already:

https://github.com/ewels/AWS-iGenomes/pull/7

Didn't know exactly where this info should be dropped, but that felt like a logical thing to do as others not using Sarek might use the files too and can find the information about where I got these files from there then, too ? 

The `.fai` files and `.dict` match - double checked that and also ran local test data (~50GB) with these change to check that everything is fine 👍

Reply:
That should all be here: https://github.com/ewels/AWS-iGenomes/pull/7